sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Metrics/GetMetricFeedbackOptions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using Azure.Core;
+
+namespace Azure.AI.MetricsAdvisor.Models
+{
+
+    /// <summary> The GetMetricFeedbackOptions. </summary>
+    public partial class GetMetricFeedbackOptions
+    {
+        internal MetricFeedbackFilter _metricFeedbackFilter;
+
+        /// <summary> Initializes a new instance of GetMetricFeedbackOptions. </summary>
+        /// <param name="metricId"> filter feedbacks by metric id. </param>
+        public GetMetricFeedbackOptions(string metricId)
+        {
+            Argument.AssertNotNullOrEmpty(metricId, nameof(metricId));
+            Guid metricIdGuid = ClientCommon.ValidateGuid(metricId, nameof(metricId));
+
+            MetricId = metricIdGuid;
+        }
+
+        /// <summary> The metric Id used to filter the feedbacks. </summary>
+        public Guid MetricId { get; }
+
+
+        /// <summary> The dimension filter. </summary>
+        public FeedbackDimensionFilter DimensionFilter { get; set; }
```

Comment:
Similarly, we should hide the `FeedbackDimensionFilter` here and expose the inner dictionary as a `DimensionKey` here. Java simply calls it `Filter`, but I think they're being a bit inconsistent with these namings for now.